substrate/primitives/application-crypto/src/ecdsa_bls381.rs

```diff
@@ -21,6 +21,13 @@ use crate::{KeyTypeId, RuntimePublic};
 use alloc::vec::Vec;
 
 pub use sp_core::paired_crypto::ecdsa_bls381::*;
+use sp_core::{
+	bls381,
+	crypto::CryptoType,
+	ecdsa, ecdsa_bls381,
+	pop::{NonAggregatable, ProofOfPossessionVerifier},
+	testing::{BLS381, ECDSA, ECDSA_BLS381},
```

Comment:
Something is extremely wrong here with these ecdsa_* stuff. These keytypes are application specific and you are just using the testing ones. This is clearly not correct. 

Keytypes are for example "BABE" or "GRANDPA" (not the real keytypes, just for illustrations)

Reply:
We just followed the convention was there for all other crypto to use app_crypto! macro: https://github.com/paritytech/polkadot-sdk/blob/61ff954f0492c4e2bd96574e44bca92edf483318/substrate/primitives/application-crypto/src/sr25519.rs#L28 put by @davxy two years ago . Perhaps someone should change all crypto types in another PR if it is not the right way? But this is not specific to the code introduced by this PR so I don't think this change should be addressed by this PR.

Reply:
@davxy found it like that already ;-P

https://github.com/paritytech/polkadot-sdk/commit/7985495b8c87b86547cd204036233f4046bcdfda

perhaps these should be moved out of testing module?